packages/ember-application/lib/system/application-instance.js

```diff
@@ -343,6 +343,15 @@ class BootOptions {
     */
     this.isInteractive = environment.hasDOM; // This default is overridable below
 
+
+    /**
+      @property _renderMode
+      @type string
+      @default false
+      @private
+    */
+    this._renderMode = options._renderMode;
```

Comment:
This was recently changed and I think the commit history of this PR is now a little inaccurate to what actually happened.  I suspect we'll sqaush soon anyways, so I'm not worried about it, but if it is a concern I can flesh out the commit history.